### ISSUE TITLE
Authentication fail

### ISSUE THREAD

**Issue by LuchiLucs** [Spotifyd/spotifyd#712]
**Description**
Running spotifyd and passing credentials with arguments works fine (between quotes). Running spotifyd from conf file doesn't work (with or without quotes): BadCredentials

**To Reproduce**
spotifyd --no-daemon -u username -p "password" 
spotifyd --no-daemon --config-path ~/.config/spotifyd/spotifyd.conf

**Expected behavior**

I would like to use the conf file

**Logs**
<details><summary>Click to show logs</summary>

  
Caught panic with message: Authentication failed with reason: BadCredentials
</details>

**Versions (please complete the following information):**

- OS: ArchLinux 5.9.1-arch1-1
- Spotifyd: 0.2.24
- cargo:

**Comment by ZweiEuro** [comment]
@LuchiLucs  Have you tried parsing a config that _only_ has pw and login ? Does it crash then? If it does this is a dupe with https://github.com/Spotifyd/spotifyd/issues/708

**Comment by LuchiLucs** [comment]
Yes, it contains all kind of characters. I don't want to change my password though. Is there a way to parse the config file correctly?

**Comment by robinvd** [comment]
@LuchiLucs we switched to TOML on the master branch (dont know if you use that version). can you try with that?

**Comment by LuchiLucs** [comment]
@robinvd I use spotifyd version 0.2.24 (spotifyd --version)

**Comment by robinvd** [comment]
@LuchiLucs unfortunately currently the TOML support in unreleased. So you have to install from source.

```
git clone https://github.com/Spotifyd/spotifyd
cd spotifyd
cargo run -- --no-deamon # or other required arguments
```

**Comment by LuchiLucs** [comment]
@robinvd Thank you. I will wait for it to be released: I have limited data usage monthly and I don't want to install/uninstall cargo/rust building dependencies for that.

**Comment by ZweiEuro** [comment]
@robinvd your guess when it will be released as the live package?

**Comment by robinvd** [comment]
@ZweiEuro i just made #714, and want to get that merged so people can read about breaking changes in new release (like TOML). It just meeds 2 approvals from maintainers. #695 would also be nice

**Comment by robinvd** [comment]
see https://github.com/Spotifyd/spotifyd/pull/716 for the version update

**Comment by strafe** [comment]
@robinvd I appreciate you're working on this, but the whole incorrect config format in the README is a nightmare for onboarding new users.

For anyone on macOS who wants to avoid the headache of the INI file, uninstall `spotifyd` if you already have it and run
`brew install spotifyd --head` (it may take a while). Now you can use the new config file format.

**Comment by robinvd** [comment]
@strafe yes i agree!

Im the feature maybe we can release faster, or show the latest release on the homepage

**Comment by slondr** [comment]
Should be resolved with v0.2.25 (#716)

Let us know if the issue persists
